config/routes.rb

```diff
@@ -7,125 +6,111 @@
   end
 
-  scope controller: 'password_management' do
+  #########################
+  #  Password Management  #
+  #########################
+  scope controller: :password_management do
     # Password management process (forgot,  change, or create password)
-    get 'forgot_password_form'
-    post 'send_reset_password_email'
-    get 'reset_password_email_sent'
-    get 'create_password_form'
-    post 'create_password'
-    get 'change_password_form'
-    post 'change_password'
-  end
-
-  scope controller: 'social_auth' do
-    get 'auth/:provider', action: :oauth_callback
+    get 'forgot_password_form', action: :forgot_password_form, as: :forgot_password_form
+    post 'send_reset_password_email', action: :send_reset_password_email, as: :send_reset_password_email
+    get 'reset_password_email_sent', action: :reset_password_email_sent, as: :reset_password_email_sent
+    get 'create_password_form', action: :create_password_form, as: :create_password_form
+    post 'create_password', action: :create_password, as: :create_password
+    get 'change_password_form', action: :change_password_form, as: :change_password_form
+    post 'change_password', action: :change_password, as: :change_password
+  end
+
+  #########################
+  #  Social Auth          #
+  #########################
+  scope controller: :social_auth do
+    get 'auth/:provider', action: :oauth_callback, as: :social_auth
     post 'auth/:provider', action: :oauth_callback
     get 'auth/:provider/callback', action: :oauth_callback
-    delete 'auth/:provider', action: :remove_auth_strategy, as: :destroy_authentication
+    delete 'auth/:provider', action: :remove_auth_strategy
     #   When you sign up with a social provider, you must confirm your info first
-    get 'confirm_your_info'
-    post 'confirm_oauth_info'
-  end
-
-  scope controller: 'sessions' do
-    get 'logout', action: :destroy
-
-    # Maintain these deprecated routes for a while until client code learns to
-    # use /login and /logout
-    get 'signout', action: :destroy
+    get 'confirm_oauth_info', action: :confirm_oauth_info
+    post 'confirm_oauth_info', action: :confirm_oauth_info
   end
 
-  mount OpenStax::Api::Engine, at: '/'
-
-  # Create a named path for links like `/auth/facebook` so that the path prefixer gem
-  #     will appropriately prefix the path. https://stackoverflow.com/a/40125738/1664216
-  # The actual request, however, is handled by the omniauth middleware when it detects
-  #     that the current_url is the callback_path, using `OmniAuth::Strategy#on_callback_path?`
-  #     So, admittedly, this route is deceiving.
-  get '/auth/:provider', to: ->(_env) { [404, {}, ['Not Found']] }, as: :auth
-
   # routes for access via an iframe
   scope 'remote', controller: 'remote' do
     get 'iframe'
     get 'notify_logout', as: 'iframe_after_logout'
   end
 
-  scope controller: 'users' do
+  scope controller: :users do
     put 'profile', action: :update
```

Comment:
I think as part of another PR, this should either move to the profile_controller or the profile "show/edit" view should go the users controllers.